### DISCUSSION TITLE
Hide non translated keys in custom locales

### DISCUSSION THREAD

**Discussion by fernion** [lingui/js-lingui#1209] [Q&A]
Hello all,

I have a project where we have custom language files for some customers. Something like:
```
📁locales/
  📁en/
    🗎 messages.json
  📁en-customer1/
    🗎 messages.json
```
**en-customer1** is configured as having **en** as fallback.

Currently with the extract command we get a lot of empty translations in the **en-customer1/messages.json** file. 
Is there a way to configure lingui so that it does not create these keys on some locales (for example, a pre-set list or the ones that have fallback)? 
The deletion of unused keys is still required though.

Basically instead of having:
```
//en/message.json
{
  "Some value": "Some value",
  "Another value": "Another value"
}
//en-customer1/message.json
{
  "Some value": "Some different value",
  "Another value": ""
}
```
we were trying to achieve:
```
//en/message.json
{
  "Some value": "Some value",
  "Another value": "Another value"
}
//en-customer1/message.json
{
  "Some value": "Some different value"
}
```

Thank you.
